db/migrate/20160715161608_uui_dto_string.rb

```diff
@@ -0,0 +1,5 @@
+class UuiDtoString < ActiveRecord::Migration
+  def change
+    change_column :users, :uuid, :string, null: false
```

Comment:
since the -idp seed users have UUID strings that are not actually valid UUIDs (not strictly base64-encoded), use a more generic `string` definition. As long as it is a MBUN we don't really care.